src/layman/user/util.py

```diff
@@ -100,3 +102,11 @@ def get_username_suggestions_from_claims(claims):
 def delete_user(username=None):
     authn_redis.delete_user(username)
     delete_whole_user(username)
+
+
+def is_allowed_to_delete_user(username=None):
+    if username is None:
+        return False
+    allowed_roles_and_users = os.getenv('GRANT_DELETE_OTHER_USER', '').split(',')
```

Comment:
We usually read environment variables only in `layman_settings.py` and save their values to same-name constants. Please do it like this, place it nearby GRANT_PUBLISH_IN_PUBLIC_WORKSPACE and GRANT_CREATE_PUBLIC_WORKSPACE constants.